cssom-1/Overview.bs

```diff
@@ -748,29 +748,23 @@ represents a style sheet as defined by the CSS specification. In the CSSOM a
 <dl>
   <dt><dfn constructor for=CSSStyleSheet>CSSStyleSheet(<var>options</var>)</dfn></dt>
   <dd>
-    When called, execute these steps:
-    <ol>
-      <li>Construct a new {{/CSSStyleSheet}} object <var>sheet</var>.</li>
-      <li>Run the steps to <a>initialize a CSSStyleSheet</a> on <var>sheet</var> given <var>options</var>.
-      <li>Return <var>sheet</var>.</li>
-    </ol>
+    When called, execute the steps to <a>create a CSSStyleSheet</a> given <var>options</var> and return the result.
   </dd>
 
-  <dt>To <dfn export>initialize a {{/CSSStyleSheet}}</dfn></dt> <var>sheet</var> given {{CSSStyleSheetInit}} <var>options</var>, run these steps:
+  <dt>To <dfn export>create a {{/CSSStyleSheet}}</dfn></dt> given {{CSSStyleSheetInit}} <var>options</var>, run these steps:
```

Comment:
Can we call this algorithm "Create a constructed CSSStyleSheet" rather than "Create a CSSStyleSheet"? We already have https://drafts.csswg.org/cssom/#create-a-css-style-sheet and I found this a bit confusing, wdyt?

Reply:
Sounds good to me -- I've pushed this change. Thanks!